share/completions/whatis.fish

```diff
@@ -1,2 +1,14 @@
-
 complete -xc whatis -a "(__fish_complete_man)"
+
+complete -c whatis -s d -l debug -d "Debug"
+complete -c whatis -s v -l verbose -d "Verbose"
+complete -c whatis -s r -l regex -d "Interpret each keyword as a regex"
+complete -c whatis -s w -l wildcard -d "The keyword(s) contain wildcards"
```

Comment:
"Allow wildcards"?

Reply:
Thanks. It did seem awkward at the time.